### ISSUE TITLE
Server crash with Forge server version 39.0.45+

### ISSUE THREAD

**Issue by philthecoder** [mrh0/buildersaddition#49]
We have a server which auto-updates forge on restart, and when it updated to 39.0.45 today, it crashed the server when a player was logging in.
The changelog for Forge mentions 
```
- 39.0.45 Remove references to the now-broken `BlockEntity#save(CompoundTag)` method (#8235)
```
and in the crash log it says:
```
java.lang.NoSuchMethodError: 'net.minecraft.nbt.CompoundTag com.mrh0.buildersaddition.tileentity.ShelfTileEntity.save(net.minecraft.nbt.CompoundTag)'
	at com.mrh0.buildersaddition.tileentity.ShelfTileEntity.m_5995_(ShelfTileEntity.java:173) ~[buildersaddition-1.18-20211208a.jar%2342!/:1.18-20211208a] {re:classloading}
	at net.minecraft.network.protocol.game.ClientboundLevelChunkPacketData$BlockEntityInfo.m_195691_(ClientboundLevelChunkPacketData.java:153) ~[server-1.18.1-20211210.034407-srg.jar%2379!/:?]
```

[crash-2022-01-20_23.09.53-server.txt](https://github.com/mrh0/buildersaddition/files/7909232/crash-2022-01-20_23.09.53-server.txt)

**Comment by philthecoder** [comment]
The host did then update it to 39.0.53 which just has immediately crashes the server on startup. I am also working with them about it. It does look like it may have corrupted the chunks with the shelftileentity on it.

**Comment by Strana** [comment]
I'm experiencing this too, I didn't know why my chunks were corrupted though, until I found this.

It should seem apparent forge update 39.0.45 has broken mods that use BlockEntity, because a chunk that I have a shelf in has been corrupted. I suggest to mod users to stay on forge 39.0.44 until a compatability comes out on the forge end, or until mods change up things.

**Comment by mrh0** [comment]
Fixed in 20220123a.
